web/local.mk

```diff
@@ -1,20 +1,22 @@
-WEB_BABELGEN:=$(patsubst %.babel,%.js,$(shell find web/gen -name '*.babel'))
-WEB_LESSGEN:=$(patsubst %.less,%.css,$(shell find web/gen -name '*.less'))
-WEB_GEN:=$(WEB_LESSGEN) $(WEB_BABELGEN)
-WEB_STATIC:=web/index.html web/res/*.css web/res/*.js
+WEB_SRC:=$(shell find web/src -type f)
+WEB_BUILD:=$(shell find web/build -type f)
```

Comment:
The web build process creates more than just `web/build/app.js`, it also creates font files `web/build/*woff` (and other dynamically generated files). I included those so they can be copied to the `build-container` for the `web` image. Or how else do they get there?